CVE-2024-5129/ubuntu1.yml

```diff
@@ -37,15 +37,18 @@
 
     - name: Install node and project deps
       shell: >
-        export NODE_PATH=/desired/install/path/node-v20.18.0-darwin-x64 &&
-        export PATH=$NODE_PATH/bin:$PATH &&
+        cd /tmp &&
+        curl -O https://nodejs.org/dist/v20.18.0/node-v20.18.0-darwin-x64.tar.gz &&
```

Comment:
Please do not pull this package from the Internet. It's already in the data folder, please install it from the file.

Reply:
just made some changes, does it look ok?